packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,103 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Gets the date format order from moment's long date format.
+   * @param {string} momentDateFormat - Moment's long date format.
+   * @return {string} - The date format order. Possible Values (MDY, DMY, YMD)
+   */
+  static getDateFormatOrder(momentDateFormat) {
+    let dateFormatOrder;
+    if (momentDateFormat === 'MM/DD/YYYY') {
+      dateFormatOrder = DateUtil.dateOrder.MDY;
+    } else if (momentDateFormat === 'DD/MM/YYYY') {
+      dateFormatOrder = DateUtil.dateOrder.DMY;
+    } else if (momentDateFormat === 'YYYY/MM/DD') {
+      dateFormatOrder = DateUtil.dateOrder.YMD;
+    }
+
+    return dateFormatOrder;
+  }
+
+  /**
+   * Returns an object consisting of date input values based on the date format.
+   * @param {string} dateOrder - String containing the date input order. Possible values (MDY, DMY, YMD)
+   * @param {string} value - The date/placeholder string.
+   * @param {string} separator - The date separator.
+   */
+  static getDateInputValues(dateOrder, value, separator) {
+    if (!value) {
+      return false;
+    }
+
+    let day;
+    let month;
+    let year;
+    const dateInputParts = value.split(separator);
+
+    switch (dateOrder) {
+      case DateUtil.dateOrder.DMY:
+        [day, month, year] = dateInputParts;
+        break;
+      case DateUtil.dateOrder.MDY:
+        [month, day, year] = dateInputParts;
+        break;
+      case DateUtil.dateOrder.YMD:
+        [year, month, day] = dateInputParts;
+        break;
+      default:
+        [year, day, month] = dateInputParts;
```

Comment:
When would it ever come here? If this is an unsupported case, should it return an empty object {} instead?

Reply:
Makes sense, so i changed it to return the object with empty strings.
https://github.com/cerner/terra-framework/pull/1208/commits/01a9f2d9f673428c5a8dce65d64256d79c440f80